src/lib/select/select.spec.ts

```diff
@@ -396,6 +395,7 @@ describe('MatSelect', () => {
         it('should open the panel when pressing a vertical arrow key on a closed multiple select',
           fakeAsync(() => {
             fixture.destroy();
+            flush();
```

Comment:
Could you elaborate on what makes these `flush` operations necessary now?

Reply:
The flush is because something is run outside the angular zone when the overlay is attached: https://github.com/angular/material2/blob/master/src/cdk/overlay/overlay-ref.ts#L304

We can attach backdrop when the select panel is opened.

Reply:
We can see how it affects google3 on presubmit